src/aux-common/Files/FileCalculations.ts

```diff
@@ -1639,6 +1639,10 @@ export function getFileConfigContexts(
     const result = calculateFileValue(calc, file, 'aux.context');
     if (typeof result === 'string' && hasValue(result)) {
         return [result];
+    } else if (typeof result === 'number' && hasValue(result)) {
```

Comment:
@shaneYeti Can you add a couple tests for this to `FileCalculationContextTests`?

Reply:
Sure I can add those in